cmd/rekor-server/app/serve.go

```diff
@@ -182,7 +183,8 @@ func init() {
 	serveCmd.Flags().Int("http-metrics-port", 2112, "HTTP port to bind metrics to")
 	serveCmd.Flags().Int("grpc-port", 3001, "GRPC port to bind to")
 	serveCmd.Flags().String("grpc-address", "127.0.0.1", "GRPC address to bind to")
-	serveCmd.Flags().Duration("timeout", 60*time.Second, "timeout")
+	serveCmd.Flags().Duration("server-timeout", 20*time.Second, "timeout settings for gRPC and HTTP connections")
+	serveCmd.Flags().Duration("log-timeout", 30*time.Second, "timeout for terminating the tiles log queue")
```

Comment:
nit: I might move this under `// tessera lifecycle configs` or rename this to mention Tessera, since there's a few other flags about logging